File: swagger/models/get_class_schedules_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .class_schedule import ClassSchedule

class GetClassSchedulesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		class_schedules
		Contains information about the class schedules.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ClassSchedules': ['class_schedules', ClassSchedule],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'class_schedules': 'ClassSchedules',
		}

	pagination_response = None
	class_schedules = None

File: swagger/Models/GetClassSchedulesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClassSchedulesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the class schedules.
	 * 
	 * @var ClassSchedule[] 
	 */
	public $classSchedules;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'ClassSchedules' => ['classSchedules', 'array', ClassSchedule::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'classSchedules' => ['ClassSchedules', 'array', ClassSchedule::class],
			];
	}
}
